### PR TITLE
add widget dashboard

### PR DESCRIPTION
**Pull request by itkg-alavieille**
https://github.com/open-orchestra/open-orchestra-model-interface/pull/112
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/425
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1044

### CODE REVIEW

File: ApiBundle/Controller/ContentController.php [hunk 1]
```diff
@@ -259,4 +259,23 @@ protected function findOneContent($contentId, $language, $version = null)
...
+     * @Config\Method({"GET"})
+     *
+     * @Config\Security("has_role('ROLE_ACCESS_CONTENT_TYPE_FOR_CONTENT')")
+     *
+     * @return FacadeInterface
+     */
+    public function listContentByAuthor($published)
+    {
+        $user = $this->get('security.token_storage')->getToken()->getUser();
+        $content = $this->get('open_orchestra_model.repository.content')->findByAuthor($user->getUsername(), $published, 10);
```
Comment (itkg-ngilain) [inline ApiBundle/Controller/ContentController.php]: The author is the person who created the content I think, not the one modifying it. So what the widget must show ? Our creations or our modifications ?

File: ApiBundle/Controller/NodeController.php [hunk 2]
```diff
@@ -131,6 +131,25 @@ public function duplicateAction(Request $request, $nodeId, $version = null)
...
+     * @Config\Method({"GET"})
+     *
+     * @Config\Security("has_role('ROLE_ACCESS_TREE_NODE')")
+     *
+     * @return FacadeInterface
+     */
+    public function listNodeByAuthor($published)
+    {
+        $user = $this->get('security.token_storage')->getToken()->getUser();
+        $content = $this->get('open_orchestra_model.repository.node')->findByAuthor($user->getUsername(), $published, 10);
```
Comment (itkg-ngilain) [inline ApiBundle/Controller/NodeController.php]: same remark here

File: BackofficeBundle/Resources/public/coffee/dashboard/DashboardWidgetListView.coffee [hunk 3]
```diff
@@ -0,0 +1,33 @@
+DashboardWidgetListView = OrchestraView.extend(
+  initialize: (options) ->
+    @options = options
+    @loadTemplates [
+      "OpenOrchestraBackofficeBundle:BackOffice:Underscore/dashboard/dashboardWidgetListView"
+      "OpenOrchestraBackofficeBundle:BackOffice:Underscore/dashboard/dashboardWidgetListContentView"
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/coffee/dashboard/DashboardWidgetListView.coffee]: dashboardWidgetListContentView -> dashboardWidgetListItemView

**Comment by itkg-nanne** [comment]
:+1:

**Pull request merged**